source/docs/structure.md

```diff
@@ -21,58 +26,52 @@ Once initialised, here's what your project folder will look like:
 
 ### app/
 
-DApp Web code goes here. See embark.json on how you customize the pipeline.
+This is where the frontend of our decentralized application lives. It doesn't matter whether you prefer using one of the amazing frameworks out there, like Angular or Vue, or if you like writing vanilla JavaScript. Embark will build your application with all the source files it can find inside `app`.
```

Comment:
Similar observation as I made in another PR. Maybe it would be better to consistently use you / your *-OR-* we / our instead of switching between them.

Reply:
Absolutely. Going with "we". Depending on the scenario, sometimes, "you" makes more sense.